app/views/posts/show.erb

```diff
@@ -15,4 +15,14 @@
       </div>
     <% end %>
   </div>
-</article>
\ No newline at end of file
+</article>
+<section class="add-comment-section">
```

Comment:
- [ ] There should be a form in the `views/comments/new.html.erb` file which accepts input for creating comments. At the moment you are missing this file. Kindly add it and put the form in the file